app/components/EulaModal.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import {
+  Image,
+  Modal,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import Markdown from 'react-native-markdown-display';
+
+import boxCheckedIcon from './../assets/images/boxCheckedIcon.png';
+import boxUncheckedIcon from './../assets/images/boxUncheckedIcon.png';
+import closeIcon from './../assets/images/closeIcon.png';
+import colors from '../constants/colors';
+import Colors from '../constants/colors';
+// import { Typography } from './Typography';
+import languages from '../locales/languages';
+import ButtonWrapper from './ButtonWrapper';
+
+export default class EulaModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalVisible: false,
+      boxChecked: false,
+    };
+  }
+
+  render() {
+    return (
+      <>
+        <ButtonWrapper
+          title={languages.t('label.launch_get_started')}
+          onPress={() => this.setState({ modalVisible: true })}
+          buttonColor={Colors.VIOLET}
+          bgColor={Colors.WHITE}
+        />
+        <Modal
+          animationType='slide'
+          transparent
+          visible={this.state.modalVisible}>
+          <View style={styles.container}>
+            <View style={{ flex: 7, padding: 25, paddingBottom: 0 }}>
+              <TouchableOpacity
+                onPress={() => this.setState({ modalVisible: false })}>
+                <Image source={closeIcon} style={styles.closeIcon} />
+              </TouchableOpacity>
+              <ScrollView>
+                <Markdown style={{ body: { color: Colors.DARK_GRAY } }}>
+                  {languages.t('markdown.eula')}
+                </Markdown>
+              </ScrollView>
+            </View>
+            <View style={styles.ctaBox}>
+              <TouchableOpacity
```

Comment:
This is a reusable component, extract into `<Checkbox />`

Use Typography (it's theme text color aware)